server/core/src/irods_file_object.cpp

```diff
@@ -156,7 +156,6 @@ namespace irods {
         repl_requested_ = _rhs.repl_requested_;
         replicas_       = _rhs.replicas_;
         in_pdmo_        = _rhs.in_pdmo_;
-        replKeyVal( &_rhs.cond_input_, &cond_input_ );
```

Comment:
Is this being handled by the base class `operator=`? If not, are there any known effects of not copying the `cond_input_` here?

Reply:
It's handled by the base class.

That line was causing the work to be done twice, which resulted in a leak.

Reply:
Great, thanks